### DISCUSSION TITLE
🙏 Got error: closed (deadline exceeded)

### DISCUSSION THREAD

**Discussion by Sunvas** [bluenviron/mediamtx#1635] [Questions]
I'm trying to use rtsp-simple-server to deliver RTSP stream from an IP camera to browser via WebRTC. On local network all works perfectly, but when I'm trying connecting via Internet - it doesn't work and console show an error:

`WAR [path cam] [rtsp source] closed (deadline exceeded)`

Can that can be solved?

**Comment by aler9** [comment]
Hello, in order to connect to a RTSP server through the internet, use the RTCP/TCP transport protocol as described here:
https://github.com/aler9/mediamtx#tcp-transport

[a comment by an automated account is not shown]
